src/constants/v2/projectPreferences.ts

```diff
@@ -0,0 +1,3 @@
+export const ProjectPreferences = {
+  STICKER_MAX: 3,
```

Comment:
**suggestion** if this isn't gonna be used anywhere, probably fine to keep this const in the component itself. wdyt?

Reply:
I'll probably use it on reading the stickers from the tx on pay feed